nuxeo-core/nuxeo-core-mimetype/src/main/java/org/nuxeo/ecm/platform/mimetype/service/MimetypeRegistryService.java

```diff
@@ -312,17 +319,8 @@ public String getMimetypeFromBlob(Blob blob) {
 
     @Override
     public MimetypeEntry getMimetypeEntryByMimeType(String mimetype) {
-        MimetypeEntry mtype = mimetypeByNormalisedRegistry.get(DEFAULT_MIMETYPE);
-        if (mimetype != null) {
-            for (Map.Entry<String, MimetypeEntry> entry : mimetypeByNormalisedRegistry.entrySet()) {
-                if (mimetype.equals(entry.getValue().getNormalized())
-                        || entry.getValue().getMimetypes().contains(mimetype)) {
-                    mtype = entry.getValue();
-                    break;
-                }
-            }
-        }
-        return mtype;
+        String normalized = getNormalizedMimeType(mimetype).orElse(DEFAULT_MIMETYPE);
```

Comment:
It seems like there is a slight behavior change: previously, if the `mimetype` parameter was `null` the method would return the `MimetypeEntry` corresponding to the default mime type.
Now it will throw an NPE.
I wonder if we shouldn't keep the previous behavior?